kubejs/server_scripts/base/trading/landingTradeIntroLine.js

```diff
@@ -456,55 +895,1043 @@ const cccIronBars = getAgreement("cccIronBars", {
+      '{BlockEntityTag:{Tanks:[{Level:{Speed:0.25f,Target:1.0f,Value:1.0f},TankContent:{Amount:8000,FluidName:"create:honey"}}]}}'
+    ),
+  ],
+  requestedItems: [
+    Item.of(
+      "supplementaries:cage",
+      '{BlockEntityTag:{MobHolder:{EntityData:{id:"minecraft:zombie"},Name:"Zombie"}}}'
+    ),
+    Item.of(
+      "supplementaries:cage",
+      '{BlockEntityTag:{MobHolder:{EntityData:{id:"minecraft:zombie"},Name:"Zombie"}}}'
+    ),
+    Item.of(
+      "supplementaries:cage",
+      '{BlockEntityTag:{MobHolder:{EntityData:{id:"minecraft:zombie"},Name:"Zombie"}}}'
+    ),
+    Item.of(
+      "supplementaries:cage",
+      '{BlockEntityTag:{MobHolder:{EntityData:{id:"minecraft:zombie"},Name:"Zombie"}}}'
+    ),
+    Item.of(
+      "supplementaries:cage",
+      '{BlockEntityTag:{MobHolder:{EntityData:{id:"minecraft:zombie"},Name:"Zombie"}}}'
+    ),
+    Item.of(
+      "supplementaries:cage",
+      '{BlockEntityTag:{MobHolder:{EntityData:{id:"minecraft:zombie"},Name:"Zombie"}}}'
+    ),
+  ],
+  title: "The Adventurers are here, it is time to dance",
+  orderedAmount: 2,
+  company: "Princess Avarusa",
+  message:
+    "They found me, I will need some more undead to deal with them, It will be worthwhile for you to help me, we can both benefit from this Honey deposit, just send me the reinforcements I need first. Make sure they are not named or anything, I need them clean",
+});
+
+const pHoneyFixed = getAgreement("pHoneyFixed", {
+  paymentItems: [
+    Item.of(
+      "createdieselgenerators:canister",
+      '{BlockEntityTag:{Tanks:[{Level:{Speed:0.25f,Target:1.0f,Value:1.0f},TankContent:{Amount:8000,FluidName:"create:honey"}}]}}'
+    ),
+  ],
+  requestedItems: [
+    Item.of(
+      "supplementaries:cage",
+      '{BlockEntityTag:{MobHolder:{EntityData:{id:"minecraft:zombie"},Name:"Zombie"}}}'
+    ),
+    Item.of(
+      "supplementaries:cage",
+      '{BlockEntityTag:{MobHolder:{EntityData:{id:"minecraft:zombie"},Name:"Zombie"}}}'
+    ),
+    Item.of(
+      "supplementaries:cage",
+      '{BlockEntityTag:{MobHolder:{EntityData:{id:"minecraft:zombie"},Name:"Zombie"}}}'
+    ),
+    Item.of(
+      "supplementaries:cage",
+      '{BlockEntityTag:{MobHolder:{EntityData:{id:"minecraft:zombie"},Name:"Zombie"}}}'
+    ),
+    Item.of(
+      "supplementaries:cage",
+      '{BlockEntityTag:{MobHolder:{EntityData:{id:"minecraft:zombie"},Name:"Zombie"}}}'
+    ),
+    Item.of(
+      "supplementaries:cage",
+      '{BlockEntityTag:{MobHolder:{EntityData:{id:"minecraft:zombie"},Name:"Zombie"}}}'
+    ),
+  ],
+  title: "Honey deposit, fixed rates",
+  orderedAmount: 0,
+  company: "Princess Avarusa",
+  message:
+    "Finally, we can excavate in peace! This was exhausting. Well, anyhow, a deal is a deal. I will pay you in honey in return for more undead to boost my ranks, it is a rare deposit, I hope you can make good use of it",
+});
+
+
+// const pAgainstAdvanturers2 = getAgreement("pAgainstAdvanturers2", {
```

Comment:
I think this stems from another issue I commented on with `Advanturers` being misspelt but then corrected in other places